include/geos/algorithm/distance/DiscreteFrechetDistance.h

```diff
@@ -0,0 +1,179 @@
+/**********************************************************************
+ *
+ * GEOS - Geometry Engine Open Source
+ * http://geos.osgeo.org
+ *
+ * Copyright (C) 2009  Sandro Santilli <strk@keybit.net>
+ *
+ * This is free software; you can redistribute and/or modify it under
+ * the terms of the GNU Lesser General Public Licence as published
+ * by the Free Software Foundation.
+ * See the COPYING file for more information.
+ *
+ **********************************************************************
+ *
+ * Last port: algorithm/distance/DiscreteFrechetDistance.java 1.5 (JTS-1.10)
```

Comment:
This isn't true, use "Last port: original work" (see src/operation/sharedpaths/SharedPathsOp.cpp for example)